core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/BitrateForwarder.java

```diff
@@ -33,26 +33,26 @@ public void onDownstreamFormatChanged(int trackType, Format trackFormat, int tra
 
     @Override
     public void onLoadStarted(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs) {
-        //TODO should we send ?
+        // TODO: should we send?
     }
 
     @Override
     public void onLoadCompleted(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs, long loadDurationMs, long bytesLoaded) {
-        //TODO should we send ?
+        // TODO: should we send?
     }
 
     @Override
     public void onLoadCanceled(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs, long loadDurationMs, long bytesLoaded) {
-        //TODO should we send ?
+        // TODO: should we send?
     }
 
     @Override
     public void onLoadError(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs, long loadDurationMs, long bytesLoaded, IOException error, boolean wasCanceled) {
-        //TODO should we send ?
+        // TODO: should we send?
     }
 
     @Override
     public void onUpstreamDiscarded(int trackType, long mediaStartTimeMs, long mediaEndTimeMs) {
-        //TODO should we send ?
+        // TODO: should we send?
```

Comment:
I personally do not like `TODO`s anyway. All of these are represented by this issue https://github.com/novoda/no-player/issues/79